### PR TITLE
chore(bal-tag): add position prop to the tag and style it in bal-card

### LINKED ISSUE #317: Bal-Tag add bestseller tag
image

### PR DESCRIPTION
**Pull request by mladenplaninicic**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: packages/components/src/components.d.ts [hunk 1]
```diff
@@ -2230,10 +2230,18 @@ export namespace Components {
           * If `true`, the element is not mutable, focusable, or even submitted with the form. The user can neither edit nor focus on the control, nor its form control descendants.
          */
         "disabled": boolean;
+        /**
+          * The font weight of the tag text.
+         */
+        "fontWeight": Props.BalFontWeight;
```
Comment (hirsch88) [inline packages/components/src/components.d.ts]: - [x] should be BalTagFontWeight

File: packages/components/src/components/bal-tag/bal-tag.sass [hunk 2]
```diff
@@ -67,17 +66,22 @@
...
     font-family: var(--bal-font-family-text)
     color: var(--bal-color-grey-inverted)
-    font-weight: $weight-bold !important
+    font-weight: $weight-bold
     display: inline-flex
     align-items: center
     gap: .25rem
+    // font weight
+    +modifier(is-regular)
+      font-weight: $weight-regular
```
Comment (hirsch88) [inline packages/components/src/components/bal-tag/bal-tag.sass]: - [x] use css variable please

**Review by hirsch88 — CHANGES_REQUESTED** [review]

**Comment by hirsch88** [comment]
now I remember why we had a small size for the tables

- [x] please add the 24px size again

**Comment by mladenplaninicic** [comment]
closes #317

**Review by hirsch88 — APPROVED** [review]

**Pull request merged**